src/quacc/calculators/espresso/core.py

```diff
@@ -0,0 +1,48 @@
+from quacc.calculators.espresso.utils import construct_namelist
```

Comment:
General comment: This pseudo-calculator seems very useful! If this is something you would like to host externally (e.g. to enable use in other projects without relying on `quacc` as a dependency), certainly feel free to abstract it away to its own package. We can add it as a dependency here in quacc. I don't think the "calculator" interface is using anything from quacc specifically aside from the YAML reader. Admittedly, this is me inquiring if there's a way to offload some of the maintenance burden on the calculator side (where my expertise is not as strong as the recipes), but I am not opposed to having it here if that's what you think is best.